assets/app/features/upgrade/controllers/upgrade-landing.controller.js

```diff
@@ -65,32 +92,31 @@
                 .getPreliminaryChecks()
                 .then(
                     //Success handler. Al precheck passed successfully:
-                    function(prechecksResponse) {
+                    function(response) {
 
-                        _.forEach(prechecksResponse.data, function(value, key) {
-                            vm.prechecks.checks[key].status = value;
+                        _.forEach(response.data, function(value, key) {
+                            vm.prechecks.checks[key].status = value.passed || !value.required;
```

Comment:
Not in the old design. Here the best we can do (without modifying the checklist directive) is treat the non-mandatory checks as passed even if they're not. Otherwise these would block the upgrade.